ksql-docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
```

Comment:
What's this dockerfile? Can we keep the old one with all the instructions on how to build and run it locally? Also this one uses the local image, which probably shouldn't be checked in.

Reply:
My bad, shouldn't haven been checked in. I thought I had specifically removed it :/